spyder_terminal/terminalplugin.py

```diff
@@ -214,7 +215,7 @@ def get_plugin_title(self):
 
     def get_plugin_icon(self):
         """Return widget icon."""
-        return ima.icon('cmdprompt')
+        return ima.icon('copywop')
```

Comment:
Please use `DollarFileIcon` for this, and remove the icons shown on the terminal tabs (if any).

Reply:
I'm not sure if the dollar file icon ($) is as global as the (>_) copywop for a terminal. If you search terminal icons it shows the second one.

Reply:
The thing is that `copywop` is too similar to the IPython console icon. Besides, we'd like to unify icons with JupyterLab:

![Selección_056](https://user-images.githubusercontent.com/365293/70364808-20870980-185c-11ea-80f8-3412d08babd5.png)

I'll do this change myself because I'm making some minor style issues too.